src/validation.h

```diff
@@ -614,6 +614,11 @@ void ResetBlockFailureFlags(CBlockIndex* pindex) EXCLUSIVE_LOCKS_REQUIRED(cs_mai
 /** The currently-connected chain of blocks (protected by cs_main). */
 extern CChain& chainActive;
 
+extern CChainState g_chainstate;
```

Comment:
This hasn't been fixed?

Reply:
Sorry - I'm marking these resolved as I rebase locally as a checklist for myself. Instead of pushing each time I rebase, I'm going to push all the changes up once I'm done (today or tomorrow).